charts/linkerd2/templates/destination.yaml

```diff
@@ -290,9 +289,12 @@ spec:
           readOnly: true
       {{ if not .Values.cniEnabled -}}
       initContainers:
-      {{- if not (contains "443" ( .Values.proxyInit.ignoreOutboundPorts | toString ))}}
-      {{- $_ := set $tree.Values.proxyInit "ignoreOutboundPorts" ( ternary "443" (printf "%s,443" (.Values.proxyInit.ignoreOutboundPorts | toString )) (empty .Values.proxyInit.ignoreOutboundPorts)) -}}
-      {{- end}}
+      {{- /*
+        The destination controller needs to connect to the Kubernetes API before the proxy is able
+        to proxy requests, so we always skip these connections. The identity controller makes no
+        other outbound connections (so it's not important to persist any other skip ports here)
+      */}}
+      {{- $_ := set $tree.Values.proxyInit "ignoreOutboundPorts" "443" }}
```

Comment:
Similar to the proxy injector we should be able to remove this. Because of the template ordering and `Linkerd-await`, the destination container will not start until the proxy is ready. I removed this and tested it locally and it works.

```suggestion
```

Reply:
I ran into problems with that -- as the proxy starts up it tries to connect to the policy controller, which hasn't started serving data yet, because it needs to connect to Kubernetes.

Reply:
Yep this seems to be templating comments not working as I thought. I was comment out this line with `#` but it seems that it will still get rendered even thought I thought it was not.